packages/web/src/common/store/cache/collections/commonSagas.ts

```diff
@@ -697,7 +696,6 @@ function* confirmDeleteAlbum(playlistId: ID, userId: ID) {
       },
       function* () {
         console.debug(`Successfully deleted album ${playlistId}`)
-        yield* put(cacheActions.remove(Kind.COLLECTIONS, [playlistId]))
```

Comment:
why is this function needed at all now?

Reply:
good q, i think just for the debug statement?